src/Header.js

```diff
@@ -33,9 +33,9 @@ class Header extends Component {
 			iconElementLeft = <IconButton onTouchTap={Header.goBackToGrid}><IconArrowBack/></IconButton>;
 		}
         if (pageLocation === "settings") {
-            iconElementRight = <IconButton href="/create/eventType"><IconAdd/></IconButton>;
+            iconElementRight = <IconButton href="settings/eventType/create"><IconAdd/></IconButton>;
         }
-        if (pageLocation === "createEventType") {
+        if (pageLocation === "create") {
```

Comment:
To only use _create_ could be difficult in the future in case we want to add other creating views.
Why didn't you use _createEventType_ as it was before?